fetch/data-urls/resources/data-urls.json

```diff
@@ -106,6 +106,9 @@
   ["data:x/x;base64;base64,WA",
    "x/x",
    [88]],
+  ["data:x/x;base64;charset=x,WA",
```

Comment:
@annevk, I'm pretty sure this is correct per https://mimesniff.spec.whatwg.org/#parse-a-mime-type and https://github.com/whatwg/fetch/pull/579, but please review. Pushed so I can see what all browsers do via w3c-test.org.